views/pages/AccountPage.jsx

```diff
@@ -75,6 +45,9 @@ class AccountPage extends Component {
       console.error(err.message)
     }
   }
+  componentWillMount() {
```

Comment:
```suggestion
  componentDidMount() {
```

Reply:
This work was updated on this pr. https://github.com/AMP-SCZ/dpdash/pull/320